uikit/src/main/java/com/midtrans/sdk/uikit/views/shopeepay/payment/ShopeePayPaymentActivity.java

```diff
@@ -0,0 +1,303 @@
+package com.midtrans.sdk.uikit.views.shopeepay.payment;
+
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AlertDialog;
+import android.text.TextUtils;
+import android.view.View;
+import android.view.View.OnClickListener;
+import android.view.ViewGroup;
+import android.view.ViewStub;
+import android.widget.RelativeLayout;
+import android.widget.RelativeLayout.LayoutParams;
+import android.widget.Toast;
+import com.midtrans.sdk.corekit.core.Logger;
+import com.midtrans.sdk.corekit.models.TransactionResponse;
+import com.midtrans.sdk.uikit.R;
+import com.midtrans.sdk.uikit.abstracts.BasePaymentActivity;
+import com.midtrans.sdk.uikit.utilities.SdkUIFlowUtil;
+import com.midtrans.sdk.uikit.utilities.UiKitConstants;
+import com.midtrans.sdk.uikit.views.shopeepay.status.ShopeePayStatusActivity;
+import com.midtrans.sdk.uikit.widgets.FancyButton;
+
+public class ShopeePayPaymentActivity extends BasePaymentActivity implements ShopeePayPaymentView {
+
+    private ShopeePayPaymentPresenter presenter;
+    private FancyButton buttonPrimary;
+    private View buttonPrimaryLayout;
+    private Boolean isAlreadyGotResponse = false, isShopeeInstalledWhenPaused;
+    private int shopeePayIntentCode, attempt;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        showProgressLayout();
+        setContentView(R.layout.activity_shopeepay_payment);
```

Comment:
Can you add prefix to the resource? it will prevent the resource being merged in the host app.
something like `activity_uikit_shopeepay_xx` or `uikit_activity_shopeepay_xx`. Last time there's a merchant that find it on their app.
Can you do that to the others resource like assets or string?

Reply:
related to shopeepay only mas?

Reply:
Yes mas, shopeepay only for now

Reply:
please check again mas